client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -540,7 +540,7 @@ export class EditorHtmlToolbar extends Component {
 						onClick={ this.openSimplePaymentsDialog }
 					>
 						<Gridicon icon="money" />
-						<span data-e2e-insert-type="payment-button">{ translate( 'Payment Button' ) }</span>
+						<span data-e2e-insert-type="payment-button">{ translate( 'Payment button' ) }</span>
```

Comment:
:disappointed: 21 existing translations will be lost with this change.
If the string must change, the following option has already been translated **19** times:
`translate( 'Insert payment button' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).